package.json

```diff
@@ -34,16 +35,26 @@
     "@radix-ui/react-slot": "^1.0.2",
     "@radix-ui/react-switch": "^1.0.3",
     "@radix-ui/react-tooltip": "^1.0.7",
+    "@tanstack/query-core": "^5.29.0",
     "@tanstack/react-query": "^5.25.0",
     "@wagmi/core": "^2.6.5",
+    "@walletconnect/ethereum-provider": "^2.12.1",
+    "@web3modal/common": "^4.1.5",
+    "@web3modal/core": "^4.1.5",
+    "@web3modal/scaffold-utils": "^4.1.5",
+    "@web3modal/siwe": "^4.1.5",
+    "@web3modal/ui": "^4.1.5",
     "@web3modal/wagmi": "^4.0.13",
+    "@web3modal/wallet": "^4.1.5",
     "async-mutex": "^0.4.1",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.0",
     "cmdk": "^1.0.0",
     "ethers": "^6.11.1",
     "fuse.js": "^7.0.0",
     "html-react-parser": "^5.1.8",
+    "js-sha3": "^0.9.3",
+    "lit-html": "^3.1.2",
```

Comment:
these are all the dependencies i had to add manually that were not found after running initial `yarn install`